napari/_qt/_tests/test_qt_notifications.py

```diff
@@ -73,3 +74,19 @@ def test_notification_display(mock_show, severity, monkeypatch):
     dialog.toggle_expansion()
     assert not dialog.property('expanded')
     dialog.close()
+
+
+@patch('napari._qt.dialogs.qt_notification.QDialog.show')
+def test_notification_error(mock_show, monkeypatch):
+    from napari.utils.settings import SETTINGS
+
+    monkeypatch.delenv('NAPARI_CATCH_ERRORS', raising=False)
+    monkeypatch.setattr(SETTINGS.application, 'gui_notification_level', 'info')
+    try:
+        raise ValueError('error!')
+    except ValueError as e:
+        notif = ErrorNotification(e)
+
+    dialog = NapariQtNotification.from_notification(notif)
+    bttn = dialog.row2_widget.children()[2]
+    assert bttn.text() == 'View Traceback'
```

Comment:
```suggestion
    bttn = dialog.row2_widget.findChild(QPushButton)
    assert bttn.text() == 'View Traceback'
    mock_show.assert_not_called()
    bttn.click()
    mock_show.assert_called_once()
```

Reply:
(slight improvement... at least confirms that clicking the button shows a dialog...).  as I'm sure you found in this case, it's tough to grab the dialog here since without a viewer instance, there is no `parent.parent()`.  We can patch that to make it easier later if necessary... but I'm happy with this test.  thanks!

Reply:
ah, findChild is useful thanks ! I was looking for something like this.